source/includes/api/_dataset.md

```diff
@@ -943,28 +1184,27 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/5306fd54-df71-4e20-8b34-2f
 }'
 ```
 
-<aside class="notice">
-    This is an authenticated endpoint!
-</aside>
+This endpoint returns a new dataset that is a replica of the dataset provided. It can take the optional query parameter `fullCloning=true` to force the cloning of the metadata.
```

Comment:
Confused as to what happens when `fullCloning=false`. What doesn't get cloned by default?

Reply:
hopefully this is clearer in the latest version of the docs